bower.json

```diff
@@ -8,5 +8,9 @@
     ".*",
     "HISTORY.md",
     "SpecRunner.html"
-  ]
+  ],
+  "dependencies": {
+    "jasmine": ">=2",
+    "jquery": ">=2"
```

Comment:
I know this is a little late to comment, but this change effectively prohibits use of Jquery 1.x series on any project dependent of this lib for testing.

A little unsure about how to solve this. Will write an issue